futurenhs.app/components/_pageTemplates/GroupMemberListingTemplate/index.tsx

```diff
@@ -93,6 +180,16 @@ export const GroupMemberListingTemplate: (props: Props) => JSX.Element = ({
                     }),
                 }
 
+                const acceptForm: FormConfig = selectForm(forms, formTypes.ACCEPT_GROUP_MEMBER)
```

Comment:
Any reason not to use useForm here?

Reply:
Can't unfortunately, it's not at top level and inside another hook - breaks the rules